src/modules/stakewise/stakewise-daemon/server/status/get-validator-statuses.go

```diff
@@ -51,12 +51,10 @@ func (c *statusGetValidatorsStatusesContext) PrepareData(data *swapi.ValidatorSt
 	bc := sp.GetBeaconClient()
 	w := sp.GetWallet()
 	nc := sp.GetNodesetClient()
-
```

Comment:
Was this your linter being overly aggressive? The newlines here were useful for organization.